<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>relatoriosimples</class>
 <widget class="QWidget" name="relatoriosimples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relat√≥rio Simples</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineNome">
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="tableResultados"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="botaoChamados">
     <property name="text">
      <string>Chamados</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="botaoPessoas">
     <property name="text">
      <string>Pessoas</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboTipo"/>
   </item>
   <item row="1" column="2">
    <widget class="QDateEdit" name="dateAbertura"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="botaoAtivos">
     <property name="text">
      <string>Ativos</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
